### ISSUE TITLE
How to handle google tag manager ?

### ISSUE THREAD

**Issue by agelospanagiotakis** [brainsum/cookieconsent#28]
What's the best strategy to handle the google tag manager cookies ?

**Comment by GamerWindow** [comment]
Have you found a solution yet? 
I am in the same situation with Matomo, looking for the best way to implement the script.

**Comment by cosminv6u** [comment]
Indeed, this needs to be extended for <noscript> as well. Facebook Pixel has <noscript> also

**Comment by tdrzewosz** [comment]
How can I block google tag manager? thanks
